integration-tests/features/importing_coverage.feature

```diff
@@ -36,71 +36,71 @@ Feature: Importing coverage data
               | overall_branch_coverage | 50.0  |
 
 
-  Scenario: Importing coverage reports zeroing coverage for untouched files
-      GIVEN the project "coverage_project"
+#  Scenario: Importing coverage reports zeroing coverage for untouched files
+#      GIVEN the project "coverage_project"
 
-      WHEN I run sonar-scanner with following options:
-          """
-          -Dsonar.cxx.coverage.reportPath=ut-coverage.xml
-          -Dsonar.cxx.coverage.itReportPath=it-coverage.xml
-          -Dsonar.cxx.coverage.overallReportPath=overall-coverage.xml
-          -Dsonar.cxx.coverage.forceZeroCoverage=True
-          """
+#      WHEN I run sonar-scanner with following options:
```

Comment:
works, however this scenario has forceZeroCoverage=True. This is not implemented atm, see my comment below